sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala

```diff
@@ -139,7 +140,13 @@ object ReloadWorkflow {
       )
     IO.write(entryPoint, depsFileContent.show)
 
-    Webpack.run(entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
+    customWebpackConfigFile match {
+      case Some(configFile) =>
+        Webpack.run("--config", configFile.getAbsolutePath, entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
```

Comment:
That would work i guess. One problem is that the custom webpack file is copied to the target dir which. Therefore the 'module' config needs to be imported using an absolute path.

Maybe a better approach would be to actually add a config setting for the modules only. This can than be used for the reload workflow but also be automatically included in the generated webpack config. A custom config file would still need to manually include the module config but it would be located in the same directory where the custom file is copied.

I imported my custom config using 
var modulesConfig = require('/home/.../project/js/webpack-reload.config.js')

modulesConfig.modules should than be merged with the configuration.

Is this the way to do it?